main/src/com/google/refine/exporters/sql/SqlCreateBuilder.java

```diff
@@ -82,13 +83,21 @@ public String getCreateSQL() {
       
                 
                 if (name != null) {
-                    if(trimColNames) {
+                	if(trimColNames) {
                         String trimmedCol = name.replaceAll("\\s", "");
-                        createSB.append( trimmedCol + " ");
-                    }else{
-                        createSB.append(name + " ");
-                    }
-                   
+                        if(dbType.equals("PostgreSQL")) {
+                    	       createSB.append("\"" +trimmedCol.replaceAll("'","''")+ "\" ");
+                    	   }else {
+                    		   createSB.append("`"+trimmedCol+"`" + " ");
```

Comment:
There are several copies of what's effectively the same code. Let's create a convenience method like `String quoteName(String name, quoteChar)` to encapsulate this.

I'm not familiar with SQL's quoting rules. Do single quotes have to be escaped (ie doubled) even though they are enclosed in double quotes?

Reply:
Yes sir single quotes have to be escaped even though they are enclosed in PostgreSQL.